crates/ra_ide_db/src/defs.rs

```diff
@@ -52,6 +52,7 @@ impl Definition {
                     let parent = id.parent_enum(db);
                     module?.visibility_of(db, &ModuleDef::Adt(Adt::Enum(parent)))
                 }
+                ModuleDef::Function(f) => Some(f.visibility(db)),
```

Comment:
we should do the same for consts and type aliases.

Reply:
Also, maybe we should move this logic into a function in `code_model` called something like `definition_visibility` :thinking:

Reply:
done it in two separate commits